fix/webs.lua

```diff
@@ -0,0 +1,60 @@
+--Cause floating webs to fall
+--By Bumber
+--@module = true
+--[====[
+
+fix/webs
+========
+Turns floating webs into projectiles, causing them to fall down to a valid surface
+Use ``fix/webs -all`` to turn all webs into projectiles, causing webs to fall out of branches, etc.
+
+]====]
+local utils = require "utils"
+
+function fix_webs(air_only)
+    if not dfhack.isMapLoaded() then
+        qerror("Error: Map not loaded!")
+    end
+    
+    local count = 0
+    for i, item in ipairs(df.global.world.items.all) do
```

Comment:
Is there a more specific item list we can use here so we can skip obviously unrelated items?

Reply:
Just from the name: `df.global.world.items.other.ANY_WEBS` might do the trick